teamcity: failed tests on master: testrace/TestTrace

The following tests appear to have failed:

[#540811](https://teamcity.cockroachdb.com/viewLog.html?buildId=540811):


```
--- FAIL: testrace/TestTrace (12.490s)
test_log_scope.go:81: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestTrace735275982
test_log_scope.go:62: use -show-logs to present logs inline
--- FAIL: testrace/TestTrace: TestTrace/Session: TestTrace/SessionDistSQL (1.450s)

--- FAIL: testrace/TestTrace: TestTrace/Session: TestTrace/SessionDistSQL: TestTrace/SessionDistSQL/TracingOn (0.760s)

--- FAIL: testrace/TestTrace: TestTrace/Session: TestTrace/SessionDistSQL: TestTrace/SessionDistSQL/TracingOn: TestTrace/SessionDistSQL/TracingOn/node-1 (0.290s)
trace_test.go:278: expected span: "session recording", got: "request range lease"
--- FAIL: testrace/TestTrace (14.580s)
test_log_scope.go:81: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestTrace735275982
test_log_scope.go:62: use -show-logs to present logs inline
--- FAIL: testrace/TestTrace: TestTrace/Session: TestTrace/SessionDistSQL (1.670s)
None
--- FAIL: testrace/TestTrace: TestTrace/Session: TestTrace/SessionDistSQL: TestTrace/SessionDistSQL/TracingOn (0.880s)
None
--- FAIL: testrace/TestTrace: TestTrace/Session: TestTrace/SessionDistSQL: TestTrace/SessionDistSQL/TracingOn: TestTrace/SessionDistSQL/TracingOn/node-1 (0.330s)
trace_test.go:278: expected span: "session recording", got: "request range lease"

```

Please assign, take a look and update the issue accordingly.

--- a/pkg/sql/trace_test.go
+++ b/pkg/sql/trace_test.go
@@ -41,6 +41,7 @@ func TestTrace(t *testing.T) {
 	// These are always appended, even without the test specifying it.
 	alwaysOptionalSpans := []string{
 		"[async] storage.pendingLeaseRequest: requesting lease",
+		"request range lease",
 		"range lookup",
 	}
 

